tcms/testcases/migrations/0007_convert_is_automated_to_boolean.py

```diff
@@ -16,6 +16,19 @@ def forward_migrate_data(apps, schema_editor):
         tc_history.save()
 
 
+def backward_restore_data(apps, schema_editor):
+    test_case_model = apps.get_model('testcases', 'TestCase')
+    historical_test_case_model = apps.get_model('testcases', 'HistoricalTestCase')
+
+    for test_case in test_case_model.objects.all():
+        test_case.new_is_automated = int(test_case.is_automated)
+        test_case.save()
+
+    for tc_history in historical_test_case_model.objects.all():
+        tc_history.new_is_automated = int(tc_history.is_automated)
```

Comment:
Same here. Follow the order of operations in this migration but in reverse order. When you start to rollback this migration, in order to return to a previous state where it is not applied, the following will happen:

1) The bool field is_automated will be renamed to bool new_is_automated
2) The int field is_automated will be added
3) RunPython will be executed with the backwards callable
4) The bool field new_is_automated will be removed
5) The transaction will complete & commit to disk.